modules/installation-creating-gcp-private-dns.adoc

```diff
@@ -25,9 +25,7 @@ have to contact Red Hat support with your installation logs.
 
 .Prerequisites
 
-* Configure a GCP account.
-* Generate the Ignition config files for your cluster.
-* Create and configure a VPC and associated subnets in GCP.
+* You have defined the variables in the _Exporting common variables_ section.
```

Comment:
FYI some variables referenced in this section, `CLUSTER_IP` and `CLUSTER_PUBLIC_IP`, are defined in the section "Creating load balancers in GCP", rather than in "Exporting common variables" section. So?